infrastructure/kube/templates/keep-client/initcontainer/provision-keep-client/provision-keep-client.js

```diff
@@ -11,10 +11,18 @@ const ethWSUrl = process.env.ETH_WS_URL
 const ethNetworkId = process.env.ETH_NETWORK_ID;
 
 // Contract owner info
-var contractOwnerAddress = process.env.CONTRACT_OWNER_ETH_ACCOUNT_ADDRESS;
-var authorizer = contractOwnerAddress
+const contractOwnerAddress = process.env.CONTRACT_OWNER_ETH_ACCOUNT_ADDRESS;
+const authorizer = contractOwnerAddress
+const purse = contractOwnerAddress;
```

Comment:
Given that it's now global constant, should we update `async function fundOperator(operatorAddress, purse, etherToTransfer)` by removing `purse` parameter from there?